docs(cpt): review (#3123)
scaleway/docs-content: observability/cockpit/troubleshooting/resetting-grafana-password-via-the-api.mdx

  paragraph: This page helps you reset your Grafana password using the API
tags: grafana password credentials
dates:
  validation: 2024-04-29
  posted: 2023-04-03
categories:
  - observability
---

> This page show you how to reset your Grafana user's password through the Scaleway API.

Author: ```suggestion
This page shows you how to reset your Grafana user's password through the Scaleway API.
```

<Macro id="requirements" />

  - A Scaleway account logged into the [console](https://console.scaleway.com)

>   - [Created](/identity-and-access-management/iam/how-to/create-api-keys/) an API key and that the API key has sufficient [IAM permissions](/identity-and-access-management/iam/reference-content/permission-sets/) to perform the actions described on this page

Reviewer: ```suggestion
  - [Created](/identity-and-access-management/iam/how-to/create-api-keys/) an API key with sufficient [IAM permissions](/identity-and-access-management/iam/reference-content/permission-sets/) to perform the actions described on this page
```

  - [Installed](https://curl.se/download.html) curl

> Open a terminal and paste the following command to retrieve your Grafana user ID:

Reviewer: Should the steps be numbered, or is it different for Troubleshooting pages?

Author: This troubleshooting only has two steps and I think we said to only number them if they had 3 or more steps
